src/frontend/js/components/SaleTunnel/AddressSelector/EditAddressFormModal.tsx

```diff
@@ -0,0 +1,69 @@
+import { Button, Modal, ModalSize } from '@openfun/cunningham-react';
```

Comment:
Theses two component, create and edit, look's really similar.
If we plane differences on interfaces / actions, it could worth it to have both of them.
But one component may be enough.

Reply:
IMO as they are already using a shared `useAddressesManagement` hooks that handles the form, adding one more level of abstraction would complicate the ready and maintenance of these components imo